README.md

```diff
@@ -36,6 +36,10 @@ It allows the management of mobile devices of your IT infrastructure, providing
 
 It provides a stand alone web based administration and an interface integrated in GLPI. The later one may be the preferred by those who use GLPI for asset management and ticketing.
 
+### Docker Environment
+
+We count with a docker environment to easily set up Flyve MDM, allowing you to have a development instance ready to work on, to know where to start, just follow our [documentation](http://flyve.org/docker-environment/howtos/).
```

Comment:
We count with a docker environment to easily set up Flyve MDM

It looks like french translated into english. I'm not sure this sentence is good. Are you sure ? If you have any doubt, then rewrite it an other way.

Reply:
Ok, I don't think it has something wrong but I'll rewrite it, just in case 👍

Reply:
In Spanish we use the same expression, but go ahead and rewrite it.